src/PIL/ImageFont.py

```diff
@@ -672,6 +675,7 @@ def getmask2(
         stroke_width=0,
         anchor=None,
         ink=0,
+        start=None,
```

Comment:
Also needs adding to the docstring.

Reply:
Done - https://pillow--6722.org.readthedocs.build/en/6722/reference/ImageFont.html#PIL.ImageFont.FreeTypeFont.getmask2